sdk/deepbook-v3/src/client.ts

```diff
@@ -149,13 +156,14 @@ export class DeepBookClient {
 		const adjusted_balance = balanceNumber / coin.scalar;
 
 		return {
+			transaction: tx,
```

Comment:
is this actually useful?  Is there ever a reason to use checkManagerBalance in a transaction that is executed?  For the methods that use devInspect, I'd remove the ability to pass in a transaction also wouldn't return it

Reply:
This endpoint wouldn't be used to combine with PTBs, but just as a readonly endpoint for accounting.